### DISCUSSION TITLE
Auth - changing toekn doesn't result in error.

### DISCUSSION THREAD

**Discussion by moritz4** [pocketbase/pocketbase#6163] [Q&A]
After logging in, I manually edit the JWT token in localStorage to another string. Then, when I make a `getFullList` request to the database, I get back an empty array, instead of authentication error (401). Why is PB still considering the jwt valid and considering me authenticated?

**Comment by ganigeorgiev** [comment]
> Why is PB still considering the jwt valid and considering me authenticated?

That's a wrong assumption. If the token is "invalid", we simply ignore it to allow user provided middlewares to load the auth state based on some other criteria (e.g. custom token, db stored session, etc.).

Or in other words, you are NOT considered authenticated and the request is resolved as if it was send without any token.
